HISTORY.md

```diff
@@ -1,5 +1,34 @@
 # Release History - open AEA
 
+## 1.22.0 (2022-11-1)
+
+AEA:
+- Updates the cert request serialisation process to maintain consistency across different operating systems
+- Updates the `get_or_create_cli_config` method to return default config instead of creating one
+- Introduces the `copy_class` utility function for testing different setup configurations
+- Updates the overridable policies for the configuration classes
+
+Packages:
+- Removes the unwanted autonomy dependency from the ledger connection
```

Comment:
can you clarify this for me?

Reply:
We had autonomy test plugin as a dependency for the ledger connection which was not needed